protocol/messenger_chats.go

```diff
@@ -487,10 +477,10 @@ func (m *Messenger) saveChat(chat *Chat) error {
 		chat.Identicon = identicon
 	}
 
-	// Sync chat if it's a new active public chat, but not a timeline chat
-	if !ok && chat.Active && chat.Public() && !chat.ProfileUpdates() && !chat.Timeline() {
-
-		if err := m.syncPublicChat(context.Background(), chat, m.dispatchMessage); err != nil {
+	// Sync chat if it's a new active public, 1-1 or group chat, but not a timeline chat
+	isPublicChat := !chat.Timeline() && !chat.ProfileUpdates() && chat.Public()
+	if !ok && chat.Active && (isPublicChat || chat.OneToOne() || chat.PrivateGroupChat()) {
```

Comment:
Looks like this condition is duplicated here and in `SyncDevices`.
Maybe create a `Chat` method?
```go
func (c *Chat) shouldBeSynced() bool  {
    isPublicChat := !chat.Timeline() && !chat.ProfileUpdates() && chat.Public()
    return chat.Active && (isPublicChat || chat.OneToOne() || chat.PrivateGroupChat())
}
```